src/main/io/osd.c

```diff
@@ -611,6 +611,8 @@ static bool osdDrawSingleElement(uint8_t item)
                 strcpy(buff, "STAB");
             } else if (FLIGHT_MODE(HORIZON_MODE)) {
                 strcpy(buff, "HOR ");
+            } else if (IS_RC_MODE_ACTIVE(BOXACROTRAINER)) {
```

Comment:
acro trainer was removed from emuflight.

Reply:
he's working off old master,  bad for merging

Reply:
or maybe we can "delete file" from the menu system?